docs/visualizing-data/build-a-map.mdx

```diff
@@ -67,42 +65,40 @@ The places data theme represents business and points of interest in the real wor
 </TabItem>
 
 <!-- Placenames -->
-
 <TabItem value="placenames" label="Placenames" default>
 The admins theme contains named localities and their administrative boundaries. To add labels for populated places to the map, we query the admins theme and calculate centroids of administrative areas.
```

Comment:
Replace "we query the admins theme and calculate centroids of administrative areas." with "we query the admins theme to get the primary name for each locality."